src/components/entityManager/EntityManager.tsx

```diff
@@ -66,8 +73,31 @@ type EntityManagerProps<E> = {
    * Display key field as a dropdown
    */
   displayKeyFieldAsDropDown?: boolean;
+
+  /**
+   * Extra property of the entity to display in the table
+   */
+  extraDisplayColumnName?: ExtraEntityColumn<E>;
+};
+
+type BasicReleaseRecipient = {
+  userFullName: string;
+  username: string;
+  enabled: boolean;
```

Comment:
Why this need to be redefined as we have already a graphql fragment - ReleaseRecipientFieldsFragment available?